tests/flow/test_temporal.py

```
@@ -0,0 +1,636 @@
+                 ( {'seconds': -15},                                                                        relativedelta(seconds=-15)),
+                 ( {'years': -2, 'months': -3},                                                             relativedelta(years=-2, months=-3)),
+                 ( {'years': -2, 'months': -3, 'days':  -4},                                                relativedelta(years=-2, months=-3, days=-4)),
+                 ( {'years': -2, 'months': -3, 'days':  -4, 'hours':   -5},                                 relativedelta(years=-2, months=-3, days=-4, hours=-5)),
+                 ( {'years': -2, 'months': -3, 'days':  -4, 'hours':   -5,  'minutes': -22},                relativedelta(years=-2, months=-3, days=-4, hours=-5, minutes=-22)),
+                 ( {'years': -2, 'months': -3, 'days':  -4, 'hours':   -5,  'minutes': -22, 'seconds': -7}, relativedelta(years=-2, months=-3, days=-4, hours=-5, minutes=-22, seconds=-7)),
+                 ( {'years': -2, 'weeks':  -1, 'hours': -5, 'minutes': -22, 'seconds': -7},                 relativedelta(years=-2, weeks=-1, hours=-5, minutes=-22, seconds=-7)),
+        ]
+
+        for map_input, expected in test_cases:
+            # construct the map string for the Cypher query
+            query = f"RETURN tostring(duration($d))"
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)